src/targets/gpu/fuse_mlir.cpp

```diff
@@ -314,6 +315,58 @@ void fuse_mlir::apply(module_pass_manager& mpm) const
 #endif
 }
 
+#ifdef MIGRAPHX_MLIR
+
+namespace {
+MIGRAPHX_PRED_MATCHER(is_supported_arch, instruction_ref)
```

Comment:
I dont think this needs to be a matcher as it is not reading anything from the graph. It can be a vanilla function and we can call it to conditionally call `match::find_matches(mpm, find_mlir_standalone_convolution_op{})`.

Reply:
Thanks! It something that we were discussing with @manupak today and came to the same conclusion. I will rework it